### PR TITLE
Delivery Project DB Stack

### PR DESCRIPTION
**Pull request by flochaz**
Creation of a DB Cluster for the delivery project

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Projects/Delivery/Db/pom.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.20</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
```
Comment (dooreaga) [inline Projects/Delivery/Db/pom.xml:93]: Just making a note for getting rid of this if we don't end up using it

File: Projects/Delivery/Db/src/test/java/com/ilmlf/delivery/db/DbTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package com.ilmlf.delivery.db;
```
Comment (dooreaga) [inline Projects/Delivery/Db/src/test/java/com/ilmlf/delivery/db/DbTest.java:1]: Did this end up working? I wasn't sure if the plan was still to keep the snapshots or move entirely to cdk diff

**Review by dooreaga — APPROVED** [review]

File: Projects/Delivery/Db/pom.xml (hunk 1 above)
Comment (flochaz) [reply Projects/Delivery/Db/pom.xml:93]: we do use it

File: Projects/Delivery/Db/src/test/java/com/ilmlf/delivery/db/DbTest.java (hunk 2 above)
Comment (flochaz) [reply Projects/Delivery/Db/src/test/java/com/ilmlf/delivery/db/DbTest.java:1]: Yes it does work now !

**Pull request merged**